Added main page for export templates (#154)
port-labs/port-docs: docs/exporters/k8s-exporter/export-templates/export-templates.md

---
sidebar_position: 3
---

# Export Templates

As shown in the [Mapping Kubernetes resources](../../../complete-use-cases/full-kubernetes-exporter.md) use case, mapping Kubernetes objects to Port is easy using Port's K8s exporter.
CRDs are also mappable using the exporter. To obtain all of the resource types you can query in your cluster, you can run:

```bash showLineNumbers
kubectl api-resources
```

For example, to figure out how to map your ArgoCD resources, you can run:

```bash showLineNumbers
$ kubectl api-resources | grep -i argo
applications                      app,apps           argoproj.io/v1alpha1                   true         Application
applicationsets                   appset,appsets     argoproj.io/v1alpha1                   true         ApplicationSet
appprojects                       appproj,appprojs   argoproj.io/v1alpha1                   true         AppProject
```

> Then, in order to map `applications`, your `config.yaml` will contain:

Reviewer: ```suggestion
Then, in order to map ArgoCD `applications`, add the matching resource kind to your `config.yaml`:
```

```yaml
- kind: argoproj.io/v1alpha1/applications
  identifier: .metadata.name
  blueprint: '"argoApp"'
  properties:
  ...
```

> We created some pre-built Port K8s exporter templates covering different applications.

Reviewer: ```suggestion
We created several pre-built Port K8s exporter templates covering different applications.
```

> Each template will include:

Reviewer: Right now this line and the previous one appear in the same line, if you want to separate them, you will need to add another blank link between therm

---
sidebar_position: 3
---

# Export Templates

As shown in the [Mapping Kubernetes resources](../../../complete-use-cases/full-kubernetes-exporter.md) use case, mapping Kubernetes objects to Port is easy using Port's K8s exporter.
It is also possible to export and map [CRDs](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) from your K8s cluster using the exporter. To obtain all of the resource types you can query in your cluster, you can run:

```bash showLineNumbers
$ kubectl api-resources
```

> For example, to figure out how to map your ArgoCD resources, you can run:

Reviewer 2: ```suggestion
For example, in order to figure out how to map your ArgoCD resources, you can run:
```
